### PR TITLE
Unconditionally migrate `AD_Org_ID` process parameter to `Search` from whatever it is now

### PR DESCRIPTION
**Pull request by TheBestPessimist**
https://github.com/metasfresh/metasfresh/issues/6298

### CODE REVIEW

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    )
+;
+
+
+-- Unconditionally update the `ad_reference_id` to `Search`
+UPDATE ad_process_para
+SET --
+    ad_reference_id = (SELECT ad_reference_id FROM ad_reference WHERE name = 'Search'),
+    UpdatedBy=0,
+    Updated=now()
```
Comment (metas-ts) [inline de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]:
```suggestion
    Updated=`2020-03-10 17:01:56.767453+01`
```
use a hardcoded date string that will be the same in the SQL and the DB

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        FROM ad_process_para
+    )
+;
+
+
+-- Unconditionally update the `ad_reference_id` to `Search`
+UPDATE ad_process_para
+SET --
+    ad_reference_id = (SELECT ad_reference_id FROM ad_reference WHERE name = 'Search'),
+    UpdatedBy=0,
```
Comment (metas-ts) [inline de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]:
```suggestion
    UpdatedBy=99,
```

use AD_User_ID 99 which is generally used for "manual" migrations

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    Updated=now()
+WHERE TRUE
+  AND ad_element_id = (SELECT ad_element_id FROM ad_element WHERE columnname = 'AD_Org_ID')
```
[the remark this answers is not in the document]
Comment (metas-ts) [reply de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]: please stick with teos suggestion to update only the record that actually need updating

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql (hunk 1 above)
Comment (TheBestPessimist) [reply de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]: done

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql (hunk 3 above)
[the remark this answers is not in the document]
Comment (TheBestPessimist) [reply de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]: updated sql

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql (hunk 2 above)
Comment (TheBestPessimist) [reply de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql]: Updated user id

**Pull request merged**